src/Integrations/OAuth2Client.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/**
+ * OAuth2Client class which handles access token connections.
+ *
+ * @package EightshiftForms\Integrations
+ */
+
+declare(strict_types=1);
+
+namespace EightshiftForms\Integrations;
+
+use EightshiftForms\Integrations\Core\HttpClientInterface;
+
+/**
+ * OAuth2Client class which handles access token connections.
+ */
+class OAuth2Client implements OAuth2ClientInterface
+{
+
+	/**
+	 * How much is hour in seconds.
+	 *
+	 * @var int
+	 */
+	public const HOUR_IN_SECONDS = 3600;
```

Comment:
This should be a defined WP constant

https://codex.wordpress.org/Easier_Expression_of_Time_Constants